decidim-assemblies/app/commands/decidim/assemblies/admin/create_assembly.rb

```diff
@@ -53,6 +53,7 @@ def assembly
             scopes_enabled: form.scopes_enabled,
             scope: form.scope,
             area: form.area,
+            parent_id: form.parent_id,
```

Comment:
You'll need to modify the `Decidim::Assemblies::AdminLog::AssemblyPresenter` to reflect this change

Reply:
Can we you the object instead of the `id`? (parent: form.parent)